homework_7/functional.py

```diff
@@ -0,0 +1,52 @@
+def func_chain(*args):
+
+    #sequential execution of all functions
+    def func(x):
+        for f in args:
+            x = f(x)
+        return x
+    
+    return func
+
+
+def sequential_map(*args):
+
+    return list(map(func_chain(*args[:-1]), args[-1]))
```

Comment:
Класс, правда, наверное, в 2 строки было бы чуть более читаемо
```suggestion
    chain = func_chain(*args[:-1])
    return list(map(chain, args[-1]))
```